Makefile

```
@@ -73,17 +75,32 @@ build-backend:
+	@gh secret set PROD_ENV_FILE < .env
 	@gh secret set DEV_ENV_FILE < .env.development
+	@gh secret set BRANCH_TEST_ENV_FILE < .env.branch-test
 	@echo "Secrets updated successfully!"
 
 # === Pull Request Creation ===
 
+ 
 # Create a pull request with the updated env secrets
 # -> This assumes you have a branch already created for the changes
-pr-create: env-update
+pr:
```

checkmake (0.2.2)

[warning] Target body for "pr" exceeds allowed length of 5 (7).

(maxbodylength)